pkg/asset/installconfig/azure/dns.go

```diff
@@ -124,10 +144,10 @@ func newZonesClient(session *Session) ZonesGetter {
 }
 
 //GetAllPublicZones get all public zones from the current subscription
-func (client *ZonesClient) GetAllPublicZones() (map[string]string, error) {
+func (client *ZonesClient) GetAllPublicZones() (map[string][]string, error) {
```

Comment:
this function is also missing the comments for the new map structure..

Reply:
Fixed.